internal/data/spicedb_test.go

```diff
@@ -48,19 +48,19 @@ func TestCreateRelationship(t *testing.T) {
 	spiceDbRepo, err := container.CreateSpiceDbRepository()
 	assert.NoError(t, err)
 
-	preExisting := CheckForRelationship(spiceDbRepo.client, "bob", "rbac/user", "", "member", "rbac/group", "bob_club")
+	preExisting := CheckForRelationship(spiceDbRepo.client, "rbac/group", "bob_club", "member", "rbac/user", "bob", "")
```

Comment:
If i'm understanding the goal correctly, this should have 2 more parameters right? For `resourceNamespace` and `subjectNamespace` so that we don't prefix the namespace

Reply:
When we create the relationship, we create a kessel relationship -- which knows about namespaces -- and we use the kessel `SpiceDbRepository` to store it, but when we want to verify what we've stored using `CheckForRelationship`, we are going direct to spicedb to see if the corresponding relationship has been created there. We expect namespace and object type to be combined into object type on the spicedb side, so we check for that.

The goal of this test (`TestCreateRelationship`), is to test if a relationship has been created correctly by kessel's spicedb repository. Ideally, or normally, we want to exercise this logic and no other logic. We *could* use another spicedb repository function, `spiceDbRepo.ReadRelationships` to read back relations for us, and that would know all about our specified namespaces; however, the problem there is that there is a chance that `CreateRelationships` and `ReadRelationships` can work together to conceal a common bug. For example, imagine if both of them simply ignored the specified namespace. We'd never know the relationship had been stored incorrectly in spicedb. By going straight to spicedb and testing for a hardcoded result we know is expected, we avoid executing flawed logic/thinking twice.

Anyway, you might have meant something else entirely, but hopefully that was useful. :)